plugins/plugin-languageserver/che-plugin-languageserver-ide/src/main/java/org/eclipse/che/plugin/languageserver/ide/util/OpenFileInEditorHelper.java

```diff
@@ -176,6 +176,15 @@ public void openLocation(String uri, LinearRange range) {
     }
   }
 
+  public void openLocation(String uri, TextRange range) {
```

Comment:
We get the uri and range from a LSP SymbolInformation object. There already is a method on this class that can open a lsp Location object, so let's not add a new method here.